src/materials/icons/layout.pug

```diff
@@ -0,0 +1,32 @@
+//- inspired by https://github.com/jonathantneal/svg4everybody#readability-and-accessibility
+
+//- Example: should read "The title"
+//- a(href="#")
+//-   +icon({name: 'arrow-right'})
+//-   | The title
+
+//- Example: should read "The title"
+//- a(href="#")
+//-   +icon({name: 'arrow-right', title: 'The title'}, { role: 'img' })
+
+
+//- Example: should read "forward" (fallback is the svg title)
+//- a(href="#")
+//-   +icon({name: 'arrow-right'})
+mixin icon(data, options)
+  - var options = helpers.mergeDefaultOptions(options, 'materials/icons');
+  - var shouldHaveTitle = options.role === 'img' && data.title
+  - var titleUniqueId = 'icon-title--' + helpers.v4()
+
+  if options.reference
+    svg(
+      class="icon",
+      role=options.role,
+      title=shouldHaveTitle && data.title,
+      aria-labelledby= shouldHaveTitle && titleUniqueId
+    )&attributes(attributes)
+      if shouldHaveTitle
+        title(id=titleUniqueId)= data.title
+      use(xlink:href="assets/icons/sprite.svg#" + data.name)
+  else
+    != helpers.inline("build/assets/icons/" + data.name + ".svg")
```

Comment:
This option is very nice in order to animate inline icons, but we cannot add a class or any other parameter to the icon. It is possible to parse the `pug` in order to add `&attributes(attributes)`

@meodai since you made the `jade-inline-file` do you know how to do it?